README.md

```diff
@@ -1 +1,39 @@
-# ToDoList
\ No newline at end of file
+# ToDoList
+A To-do-ist javascript app buit using webpack and es6.
+
+Prerequisties
+HTML
+CSS
+JavaScript
+Wepack
+
+Getting Started
+Make sure you have node installed on your local machine.
+
+step 1: Clone this repo locally using git in the command line and the following command
+
+git clone https://github.com/asnganje/ToDoList.git
+
+Step 2: run npm install to download the necessary modules.
+
+step 3: run npm start to load the project in your broswer.
+
+
+Author  👤 Abdulrahman Nganje
+
```

Comment:
- [ ] [OPTIONAL] Please consider making your README more professional as it's the first thing people see about your project and can make your work stand out. You can use this [template](https://github.com/microverseinc/readme-template).